c-(programming-language)/c-quiz.md

```diff
@@ -1070,3 +1070,28 @@ int main() {
 - [ ] 4
 - [ ] 2
 - [ ] 1
+
+
+ #### Q70. What will be the output of the following code snippet?
+ #include <stdio.h>  
+void change(int,int);  
+int main()  
+{  
+    int a=10,b=20;  
+    change(a,b); //calling a function by passing the values of variables.  
+    printf("Value of a is: %d",a);  
+    printf("\n");  
+    printf("Value of b is: %d",b);  
+    return 0;  
+}  
+void change(int x,int y)  
+{  
+    x=13;  
+    y=17;  
+}  
+
+output --
+[1.]10,20
+[2.]10,10
+[3.]20,20
+[4.]20,10
```

Comment:
this is not typical format for quiz answers, please take a look at examples, thanks